tee-worker/client-api/parachain-api/prepare-build/interfaces/vc/definitions.ts

```diff
@@ -22,6 +22,7 @@ export default {
                 A20: "Null",
                 Oneblock: "OneBlockCourseType",
                 GenericDiscordRole: "GenericDiscordRoleType",
+                WeirdoGhostGangHolder: "Null",
```

Comment:
Please beware we should apply https://github.com/litentry/litentry-parachain/pull/2275/files#diff-f5f0c9b0e57eac75b915382530a5cd4b938ded86e6f53aca92755be1b9f151b2 if we change the index of `WeirdoGhostGangHolder` to 18